docs/help.rst

```diff
@@ -0,0 +1,97 @@
+also means that we need to learn new tools and new ways of thinking, which can be a bit overwhelming, to the point that
+we start relying more and more on other researchers. In the end, we might feel like we loose our independence:
+
+    | And now my life has changed in oh so many ways
+    | My independence seems to vanish in the haze
+    | But every now and then I feel so insecure
+    | I know that I just need you like I've never done before
+
+    -- The Beatles, Help!
+
+We can continue with the song, but we think you get the point. The point is that we will need help, and we will need
+to ask for it. We will need to ask for help from our colleagues, from our supervisors, from our friends.
+
+We are a small team of researchers and developers, and we are not experts in *everything at once*. Each one of us has
+a specific expertise, and we are trying to use this expertise to create Junifer. When we thought of Junifer, we thought
+about researcher's problems, and try to come up with the best way to help them, by building a tool that is easy to use,
+that is easy to learn, and that is easy to understand. Most importantly, we are here to help. We are here to help you
+and your research.
+
+If you have any questions, if you have any problems, if you have any suggestions, please do not hesitate to contact
+us. We will be happy to help you, and we will be happy to hear from you.
+
+Seems nice, no? But we have one condition: **helps us help you**.
+
+We are not mind readers. We cannot know what you are thinking, and we cannot know what you are trying to do. Unless you
+tell us. **The more detailed explanation you give us, the faster we can help you**. So we have opened several
+communication channels, so that you can contact us in the way that is most convenient for you.
+
+Some people prefer to **write, in detail, with code and figures**. If you are one of those, use the
+`junifer Discussions`_. site on GitHub. This is a place where you can ask questions, and where you can discuss topics
+such as potential new features, or potential new methods.
+
+Some other people does **not have a clear idea of what they want**, but they know that they need help. This is not a
+problem, but it is a bit more difficult to help. Since it will require more frequent interactions to try to understand
+what you are trying to do, we have the `junifer matrix channel`_ in which you can chat with us and other junifer users.
+
+Finally, some people **prefer to not even leave trace of their issue in written**. If you are one of those, you might
+want to join our *office hours*. Given that our agenda might vary, office hours will be announced on the
+`junifer matrix channel`_ chat. Feel free to join and just *listen in* if you are too shy to write.
+
+In short, these are the 3 communication channels to get help:
+
+#. Discussions on Github (`junifer Discussions`_):
```

Comment:
The + and - might be a bit confusing for users to understand and I would lean towards having "Pros" and "Cons" explicitly mentioned and divided for each point.